tests/unit/test_grouped_query_attention.py

```diff
@@ -0,0 +1,78 @@
+import torch
+
+from transformer_lens.components import Attention, GroupedQueryAttention
+from transformer_lens.HookedTransformerConfig import HookedTransformerConfig
+
+
+def test_grouped_query_attention():
```

Comment:
Adding a docstring or more descriptive test name would be good! General idea makes a lot of sense though - thanks for adding this!